### ISSUE TITLE
First resubmission to do

### ISSUE THREAD

**Issue by davharris** [weecology/bbs-forecasting#225]
Assuming we merge https://github.com/weecology/bbs-forecasting/pull/223 (or something similar), I think this is a complete list of what's left. I'll try to get them all done (or at least answered) by the end of the day.

**Ward**

- [x] revisit #206?, deal with in #236 

- [x] Line 163-164: again, I think equations could really help here to describe the baseline models. 

- [x] Line 266: awesome to include a short vignette with example data from this project that is more easily accessible to readers without having to install Jupyter/R packages/etc, #229 

- [x] Line 249: Can you elaborate on what you did with the deviance? #235 

- [x] Line 329 – maybe reword, you use forecast 3x in this sentence, #230 

**Reviewer 2**

- [x] 25-29: rather than relay the type of information that the paper will provide, the authors should summarize their results and findings

- [x] 57-58: Also, the authors exclude the many deeper-time hindcasting studies that have been performed on these types of models

- [x] 466: only other mention of future forecasts… not sure why this is included in this paper.

**Comment by sdtaylor** [comment]
The first one for reviewer 2 was done in #222, it just needs an answer in the reply.   

We also need to address the 2nd  part of this one. Which we talked about putting some of the Blois citations in and making the wording about biodiversity forecasting more general. 

57-58: this is not the definition of hindcasting. This is the definition of testing. Also, the authors exclude the many deeper-time hindcasting studies that have been performed on these types of models

**Comment by davharris** [comment]
Thanks @sdtaylor!

**Comment by sdtaylor** [comment]
I found a cool website to do diffs and have compared our submitted version with the current HEAD
http://www.mergely.com/WRdLphUr/

**Comment by davharris** [comment]
Wow, that's fantastic. Adding it to my bookmarks.

**Comment by sdtaylor** [comment]
reviewer 2 stuff in #226 and #227

**Comment by ethanwhite** [comment]
We'll probably need to provide a tracked changes version of the ms as part of the resubmission. I've used Tim Poisot's method for this in the past: http://timotheepoisot.fr/2014/07/10/markdown-track-changes/

**Comment by ethanwhite** [comment]
You can also get a full diff like the nice one @sdtaylor made directly through github using `compare`: https://github.com/weecology/bbs-forecasting/compare/initial-submission...master

Which lets you do the kinds of rich diffs which are really useful for prose:
https://github.com/weecology/bbs-forecasting/compare/initial-submission...master?short_path=2112e0d#diff-2112e0db2b5026e7808dfd94ee62b151

**Comment by ethanwhite** [comment]
Anyone working on the "Line 266" comment? If not I can PR a response.

**Comment by sdtaylor** [comment]
nope

**Comment by davharris** [comment]
That would be great, @ethanwhite!

**Comment by ethanwhite** [comment]
I'll take Line 329 next barring objections.

**Comment by davharris** [comment]
@ethanwhite 👍 . Thanks!

**Comment by davharris** [comment]
I'll take 163-164 next

**Comment by sdtaylor** [comment]
do we need a Dear Editor paragraph for the reply?

**Comment by davharris** [comment]
I think we're all set, once the current PRs get finished up and merged.  I'll work on uploading all the new figures etc. in the mean time.  Thanks both of you for your hard work on this!

**Comment by sdtaylor** [comment]
final tasks in #237, #238, #241, and #242
